api/XRPermissionStatus.json

```diff
@@ -6,7 +6,8 @@
         "spec_url": "https://immersive-web.github.io/webxr/#xrpermissionstatus",
         "support": {
           "chrome": {
-            "version_added": false
+            "version_added": false,
+            "notes": "See <a href='https://crbug.com/b/1060163'>bug 1060163</a>."
```

Comment:
Does this 404 just for me?
https://crbug.com/1060163 seems to work.

Reply:
```suggestion
            "notes": "See <a href='https://crbug.com/1060163'>bug 1060163</a>."
```

👀